recipe/meta.yaml

```diff
@@ -25,11 +27,11 @@ requirements:
     - python
     - pip
     - numpy
-    - libtiff
+    - libtiff <=4.3.0
```

Comment:
Shouldn't these be `<4.4` so that all versions of 4.3.x are included?

Reply:
For me this recipe only support 4.3.0 4.2.0 4.1.0 and below.
I checked and there is no 4.2.x nor 4.1.x but if there is a 4.3.1 it will not work.

Reply:
(The file `libtiff/tiff_h_4_3_0.py` must match the exact version)

Reply:
Oh really? For some reason I thought it was more flexible than that. Sounds good then.